R/search.R

```diff
@@ -12,9 +12,6 @@ Search_ <- function(.obj = "", body = list(), params = list(), ...){
   } else {
     .obj
   }
-  if (as.character(body) == "[]") {
```

Comment:
dont remember - seems like it should be removed, do that if you can or I can do later